src/model/ai/evaluation/GameOverEvaluator.java

```diff
@@ -0,0 +1,33 @@
+package model.ai.evaluation;
+
+import model.Board;
+import model.PieceColor;
+
+public class GameOverEvaluator extends BoardEvaluator {
+
+    public GameOverEvaluator() {
+        super();
+    }
+
+    public GameOverEvaluator(double weight) {
+        super(weight);
+    }
+
+    @Override
+    public double evaluateBoard(Board theBoard, PieceColor color) {
+        double value = 0.0;
+
+        PieceColor winner = theBoard.determineWinner();
+
+        if (winner == null) {
+            return value;
+        }
+
+        if (winner == color) {
+            return 1.0 * this.weight;
+        }
+
+        return -1.0 * this.weight;
```

Comment:
I think this should be structured into an if-else if-else block rather than trailing ifs.